File: hw5/src/main/java/ru/kpfu/itis/genatulin/hw5/components/Calculator.java
package ru.kpfu.itis.genatulin.hw5.components;

import org.springframework.stereotype.Component;

@Component
public class Calculator {
    public Integer add(Integer arg1, Integer arg2) {
        return arg1 + arg2;
    }

    public Integer subtract(Integer arg1, Integer arg2) {
        return arg1 - arg2;
    }

    public Integer multiply(Integer arg1, Integer arg2) {
        return arg1 * arg2;
    }

    public Integer divide(Integer arg1, Integer arg2) {
        return arg1 / arg2;
    }
}

File: hw5/src/main/kotlin/ru/kpfu/itis/genatulin/hw5/Calculator.kt
package ru.kpfu.itis.genatulin.hw5

import org.springframework.stereotype.Component

@Component
class Calculator {
    fun add(arg1 : Int, arg2 : Int) : Int {
        return arg1 + arg2
    }

    fun subtract(arg1 : Int, arg2 : Int) : Int {
        return arg1 - arg2
    }

    fun multiply(arg1: Int, arg2: Int) : Int {
        return arg1 * arg2
    }

    fun divide(arg1: Int, arg2: Int) : Int {
        return arg1 / arg2
    }
}